### ISSUE TITLE
Document limitations of entrypoint.sh

### ISSUE THREAD

**Issue by Krigpl** [golemfactory/clay#4188]
`base.Dockerfile` uses `entrypoint.sh` script as the entrypoint which is doesn't always work as expected.
`docker run golemfactory/base:1.4 pwd` works as expected while
`docker run golemfactory/base:1.4 python3 --version` doesn't because command is not a single argument. Current workaround is to run it as
`docker run golemfactory/base:1.4 "python --version"`.
This is something that should be documented in the `entrypoint.sh` itself and in the future app developer guide as well.

**Comment by Krigpl** [comment]
No longer relevant after #4242
